src/siteConfig.js

```diff
@@ -2,8 +2,12 @@
  * Copyright (c) Facebook, Inc. and its affiliates.
  */
 
-export const siteConfig = {
-  editUrl: 'https://github.com/reactjs/react.dev/tree/main/src/pages',
+exports.siteConfig = {
+  // --------------------------------------
+  // Translations should replace these lines:
+  languageCode: 'en',
+  hasLegacySite: true,
```

Comment:
Translators should change these two lines above ^^

Reply:
(To clarify, we'll probably do this ourselves though as we're going through fixing the syncs.)